pkg/build/gobuild.go

```diff
@@ -727,19 +729,13 @@ func (g *gobuild) buildOne(ctx context.Context, refStr string, base v1.Image, pl
 		cfg.Config.Labels[k] = v
 	}
 
+	cfg.Created = g.creationTime
```

Comment:
Should this only be if `g.creationTime != empty` like before?

Reply:
I wanna say yes.